DigitalExcellence/dex-frontend
# Feature/removable highlights

In this commit I:
- Added the functionality that gives the user the ability to delete the highlights
- Added the modal to select the highlights you want to delete and confirm that you want to delete them

## Description
Added the functionality to delete one/multiple highlight(s), and implemented a modal where the user can select the highlights to be removed, and confirm that he actually wants to remove these.

## Type of change


## Checklist


## Steps to Test or Reproduce

Ensure that you are on the following branch for the backend: https://github.com/DigitalExcellence/dex-backend/pull/175

Also make sure to delete your existing database. This will force the application to seed your database again with the correct users/permissions

1. Log in using bob-bob
2. Go to the overview page of a highlighted project
3. Click on the delete button
4. Select the highlights that you want to remove
5. Click on save
6. The selected highlight(s) should be removed

## Link to issue

Closes: #180
